### DISCUSSION TITLE
confusion in the code?

### DISCUSSION THREAD

**Discussion by maxime-baclac** [carj/pyPreservica#18] [General]
![image](https://github.com/carj/pyPreservica/assets/168654656/b9bd1614-7b5e-4e41-b287-434364544c36)

so in the example above... there is a call to "entity.update..." at the end...
But I think it should be "client" instead... since in most example the entityClient is defined as client.
Entity is never referenced in the example.

Just confirming :)

**Comment by carj** [comment]
Good spot, the example is wrong. The name of the client is "client" 

I have fixed the documentation.

**Comment by maxime-baclac** [comment]
Cool stuff thanks!

**Reply by carj** [reply]
Done.

![image](https://github.com/carj/pyPreservica/assets/741026/5a4ec215-d5b6-4c4c-8366-1b806766f81a)
